mantidproject/mantid
# ISIS SANS Usability Fixes

**Description of work.**
Contains a number of usability fixes for the ISIS SANS Interface, as described in the testing section

**To test:**

- Open the SANS interface
- Single click any box in the table to edit it
- Attempt to tab over
- Downloading the ISIS Sample data, and extract the LOQ folder
- Open the masking file
- Re-Open the masking file and check your previous file was selected
- Do the same for the batch file

----

- Check settings -> General and Q, Wavelength tabs visually look correct 
- Check fields are still populated correctly (it would throw a runtime error if a field on the view is broken)
- Change the wavelength box to RangeLin or RangeLog
- Enter something comma separated such as `1,4,5,6` 
- Rerun using the batch file to see if these are now accepted

Fixes #26808 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
